frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -94,16 +109,28 @@ export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
       {moreLink ? (
         <FlexItem>
           {moreLink.onClick ? (
-            <Button onClick={moreLink.onClick} isInline variant="link">
+            <Button
+              onClick={moreLink.onClick}
+              isInline
+              variant="link"
+              data-test={`item ${moreLink.id}`}
```

Comment:
My first thought was that this should be a separate id from the list items.

Reply:
Wouldn't you describe the UI that each column contains three links? The idea here is that a test can assert for 3 links (items).

Reply:
In this case I have no strong opinion. It comes down to how we want to test it.